PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -229,7 +226,17 @@ void ScintillaEditView::init(HINSTANCE hInst, HWND hPere)
 	execute(SCI_SETMARGINMASKN, _SC_MARGE_FOLDER, SC_MASK_FOLDERS);
 	showMargin(_SC_MARGE_FOLDER, true);
 
-	execute(SCI_SETMARGINMASKN, _SC_MARGE_SYBOLE, (1<<MARK_BOOKMARK) | (1<<MARK_HIDELINESBEGIN) | (1<<MARK_HIDELINESEND) | (1<<MARK_HIDELINESUNDERLINE));
+	execute(SCI_SETMARGINMASKN, _SC_MARGE_SYMBOL, (1 << MARK_BOOKMARK) | (1 << MARK_HIDELINESBEGIN) | (1 << MARK_HIDELINESEND) | (1 << MARK_HIDELINESUNDERLINE));
+
+	execute(SCI_SETMARGINMASKN, _SC_MARGE_CHANGEHISTORY, (1 << SC_MARKNUM_HISTORY_REVERTED_TO_ORIGIN) | (1 << SC_MARKNUM_HISTORY_SAVED) | (1 << SC_MARKNUM_HISTORY_MODIFIED) | (1 << SC_MARKNUM_HISTORY_REVERTED_TO_MODIFIED));

```

Comment:
@donho you need to clear the mask from the margin when hiding the margin otherwise the marks once in the margin are show as background in the text.  The same goes for bookmarks.

1) Set a bookmark
2) Settings=>Preferences=>Margins/Border/Edge=> uncheck "Display bookmark"

watch the line background turn the color of the bookmark.  With Change History, it's not immediately noticeable, but when you open a new doc and start typing, you see the background. 

These lines are probably OK here, but you'll need to add:

```C++
execute(SCI_SETMARGINMASKN, _SC_MARGE_SYMBOL, 0)
execute(SCI_SETMARGINMASKN, _SC_MARGE_CHANGEHISTORY, 0)
```

when disabling / hiding the margins.